app/src/org/commcare/xml/AndroidTransactionParserFactory.java

```diff
@@ -87,7 +88,13 @@ public void initCaseParser() {
             @Override
             public TransactionParser<Case> getParser(KXmlParser parser) {
                 if (created == null) {
-                    created = new AndroidCaseXmlParser(parser, tallies, true, sandbox.getCaseStorage(), generator);
+                    created = new AndroidCaseXmlParser(parser, tallies, true, sandbox.getCaseStorage(), generator) {
+
+                        @Override
+                        public void onIndexDisrupted(String caseId) {

```

Comment:
if you put
```
cross-request: https://github.com/dimagi/commcare/pull/265
```
in the description of the PR then jenkins will run the tests with the correct `commcare` branch